ast/desugar/Desugar.cc

```diff
@@ -1929,6 +1929,7 @@ ExpressionPtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what)
             },
             [&](parser::Return *ret) {
                 if (ret->exprs.size() > 1) {
+                    auto arrayLoc = ret->exprs.front()->loc.join(ret->exprs.back()->loc);
```

Comment:
This is the kind of change that would be nice to land as a separate change, with tests, because it _is_ possible to write tests for this behavior absent the code action changes:

[View on sorbet.run →](https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Asig%20%7B%20returns%28Integer%29%20%7D%0Adef%20example1%0A%20%20return%20''%0Aend%0A%0Asig%20%7B%20returns%28T%3A%3AArray%5BInteger%5D%29%20%7D%0Adef%20example2%0A%20%20return%20''%2C%201%0Aend%0A%0Asig%20%7B%20returns%28Integer%29%20%7D%0Adef%20example3%0A%20%20''%0Aend%0A%0Asig%20%7B%20returns%28Integer%29%20%7D%0Adef%20example4%0A%20%20if%20%5Btrue%2C%20false%5D.sample%0A%20%20%20%200%0A%20%20else%0A%20%20%20%20''%0A%20%20end%0Aend)